src/Module/Notifications/Notification.php

```diff
@@ -78,7 +78,8 @@ public static function rawContent(array $parameters = [])
 
 		if (DI::args()->get(1) === 'mark' && DI::args()->get(2) === 'all') {
 			try {
-				$success = DI::notify()->setSeen();
+				DI::notification()->setAllSeenForUser(local_user());
+				$success = DI::notify()->setAllSeenForUser(local_user());
```

Comment:
what do we do if the method above worked, but this one don't .. It would be "partly" successful ;-)
Maybe we do need a "transactional" behavior for such cases?

Reply:
Better than "transactional", it's a "transitional" behavior, @annando said that either `notify` or `notification` is going away, can't remember which. In the mean time, there will be some parallel calls.

Reply:
The question is are we prepared in case we do need more than one list of entities to be saved at one call and what's happening in case of failure?

Reply:
Not in this case. The `notify` call won't be performed if the `notification` call fails and throw an Exception.